frontends/election-manager/src/components/full_test_deck_tally_report_button.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useContext, useEffect } from 'react';
```

Comment:
This file is reasonably well covered by `app.test.tsx`, but I'm not sure how much of the actual behavior it's testing. For example, the tests seem to pass even if I comment out the bodies of `disablePrinting` and `enablePrinting`. Perhaps add a unit test that can test those more specific things?